Makefile

```diff
@@ -22,6 +23,14 @@ format: black
 	isort $(CODE_DIRS)
 	npx prettier --write $(CONTRACTS)
 
+difflint:
+	$(eval FILES := $(shell git status --porcelain | grep --color=never -E $(DIFFLINT_RE) | awk '{print $$NF}'))
```

Comment:
Why not use `git ls-files -m`?

Reply:
Took a look on that and would be nice to use it, however it shows only modified files, while added files are also needed in my opinion.